source/sass/donate.scss

```diff
@@ -0,0 +1,30 @@
+@tailwind base;
```

Comment:
Most stylings are applied on DOM elements directly using Tailwind classes. This SCSS file is still needed to handle things that Tailwind can't easily do.